lib/std/json.zig

```diff
@@ -1624,6 +1658,11 @@ pub fn parseFree(comptime T: type, value: T, options: ParseOptions) void {
                 parseFree(arrayInfo.child, v, options);
             }
         },
+        .Vector => |vectorInfo| {
```

Comment:
could combine this with the `Array` case above.

Reply:
I tried `.Array, .Vector => |info| {`).
and got an `error: capture group with incompatible types'.